### DISCUSSION TITLE
State of FreeBSD support

### DISCUSSION THREAD

**Discussion by MrSuicideParrot** [minio/minio#15172] [Q&A] [answered]
Hi!
I was looking to install minio on freebsd. However, I noticed that you don't have any official release in your [release server](https://dl.min.io/server/minio/release/), the last release for freebsd was made on December 2021. 

I saw this comment stating that there is no official support (https://github.com/minio/minio/pull/8876), but at that time there were releases for FreeBSD.

So, nowadays what is the current position of minio about FreeBSD? There won't be further releases for FreeBSD?

**Comment by klauspost** [comment, marked as answer]
> So, nowadays what is the current position of minio about FreeBSD?

There is no active support, but as OSS we of course accept patches if something should break. But for the moment it is provided "as-is" with no official builds.

If we get commercial requests or someone from the community would take on that responsibility we might consider official support. If you are interested in that, write `sales@min.io` to discuss your case.
